modules/nodes-cluster-enabling-features-about.adoc

```diff
@@ -24,7 +24,7 @@ The following Technology Preview features are enabled by this feature set:
 ** Shared Resources CSI Driver in OpenShift Builds. Enables the Container Storage Interface (CSI). (`CSIDriverSharedResource`)
 ** Swap memory on nodes. Enables swap memory use for {product-title} workloads on a per-node basis. (`NodeSwap`)
 ** OpenStack Machine API Provider. This gate has no effect and is planned to be removed from this feature set in a future release. (`MachineAPIProviderOpenStack`)
-** Insights Operator. Enables the Insights Operator, which gathers {product-title} configuration data and sends it to Red Hat. (`InsightsConfigAPI`)
+** Insights Operator. Enables the `InsightsDataGather` CRD, which allows users to configure some Insights data gathering options. The feature set also enables the `DataGather` CRD, which allows users to run Insights data gathering on-demand. (`InsightsConfigAPI`)
```

Comment:
* Since this appears to be the first instance of "CRD" in the module, suggest spelling it out initially
* "on demand" when an adverb (per ISG)
```suggestion
** Insights Operator. Enables the `InsightsDataGather` custom resource definition (CRD), which allows users to configure some Insights data gathering options. The feature set also enables the `DataGather` CRD, which allows users to run Insights data gathering on demand. (`InsightsConfigAPI`)
```